precompile/allowlist/config.go

```diff
@@ -37,6 +39,39 @@ func (c *AllowListConfig) Configure(chainConfig precompileconfig.ChainConfig, pr
 	return nil
 }
 
+func (c *AllowListConfig) packAddresses(addresses []common.Address, p *wrappers.Packer) error {
```

Comment:
do we need these to be a receiver method (with `c`)? I think these can be good util functions in precompile/contract/utils.

Reply:
Fixed in 91bdff51